.circleci/config.yml

```diff
@@ -186,6 +186,24 @@ jobs:
             --justfile ../../../single.just \
             simulate
 
+  just_simulate_eth-mmz-002-key-handover:
+    docker:
+      - image: <<pipeline.parameters.ci_builder_image>>
+
```

Comment:
Please remove this blank line. #279 very recently moved this file away from that strange whitespace convention.

```suggestion
```